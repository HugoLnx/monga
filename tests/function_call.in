int main() {
	a(3, "asx", 4.52);
	b();
}