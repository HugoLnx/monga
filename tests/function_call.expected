program:
  function:main,259,0
    block:
      statements:
        statement:
          function call:a
            exp list:
              exp:0,3
              exp:4,"asx"
              exp:3,4.52000
        statement:
          function call:b
